css-bootstrap/styles/sass/styles.scss

```diff
@@ -0,0 +1,93 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: #FFFFFF;
+}
+
+.prueba {
+  width: 80vw;
+}
+
+.overflow-x-auto::-webkit-scrollbar {
+    display: none;
+}
+
+.text-transparent-bgclip {
+  background: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.text-fill-black {
+  -webkit-text-fill-color: #1C1C1C;
```

Comment:
can't we use the variable we defined here for the color?

Reply:
As we are overriding the property that's in the class text-transparent-bgclip: -webkit-text-fill-color: transparent, we need this class with the same property name.
Attach images bellow:

![Screen Shot 2023-03-01 at 10 34 41](https://user-images.githubusercontent.com/92554753/222153845-7898c714-a3f3-4758-9db1-e8b7175753ff.png)
<img width="712" alt="Screen Shot 2023-03-01 at 10 34 59" src="https://user-images.githubusercontent.com/92554753/222153889-f8d6764c-aec5-433e-8daf-34372330ad5a.png">